website/project/views/node.py

```diff
@@ -735,6 +736,13 @@ def _view_project(node, auth, primary=False,
             'institutions': get_affiliated_institutions(node) if node else [],
             'has_draft_registrations': node.has_active_draft_registrations,
             'is_preprint': node.is_preprint,
+            'has_moderated_preprint': node.has_moderated_preprint,
+            'preprint_state': node_linked_preprint.reviews_state if node.has_moderated_preprint else '',
+            'preprint_word': node_linked_preprint.provider.preprint_word if node.is_preprint else '',
+            'preprint_provider': {
+                'name': node_linked_preprint.provider.name,
+                'workflow': node_linked_preprint.provider.reviews_workflow
+            } if node.is_preprint else {},
```

Comment:
Could these three check whether `node_linked_preprint` is null, instead of `node.has_moderated_preprint` and `node.is_preprint`?